homeassistant/components/bang_olufsen/strings.json

```diff
@@ -1,19 +1,17 @@
 {
   "config": {
-    "abort": {
-      "already_configured": "[%key:common::config_flow::abort::single_instance_allowed%]",
-      "already_in_progress": "[%key:common::config_flow::abort::already_in_progress%]",
+    "error": {
       "api_exception": "[%key:common::config_flow::error::invalid_host%]",
       "client_connector_error": "[%key:common::config_flow::error::invalid_host%]",
```

Comment:
Is the error message "Invalid hostname or IP address" really good for these issues?

Reply:
No it is not. I've changed it to "Failed to connect" from `string.json`.